.gitignore

```diff
@@ -24,3 +24,4 @@ openqa.log
 t/data/openqa/factory/hdd/00099963-hdd_image2.qcow2
 t/data/openqa/factory/hdd/hdd_image.qcow2
 t/diagram-v*.png
+.directory
```

Comment:
where is .directory created? I guess it is by your desktop environment and not specific to openQA so IMHO this should not be ignored in general